src/go/rdctl/pkg/autostart/autostart_darwin.go

```diff
@@ -1,9 +1,50 @@
 package autostart
 
 import (
-	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
 )
 
+const launchAgentFileContents = `<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>io.rancherdesktop.autostart</string>
+    <key>ProgramArguments</key>
+    <array>
+        <string>/Applications/Rancher Desktop.app/Contents/MacOS/Rancher Desktop</string>
```

Comment:
This will need to actually refer to the application directory.
(I dumped the app on the desktop to try it out, and it totally didn't work…)

Reply:
I wonder if we should re-write the auto-start settings every time the app is launched. That way it will fix itself if the user moves the app around.

It is still possible for the user to move the app and log out, and on the next login, the auto-start is broken. But once the user launches the app manually, the auto-start script would be fixed automatically.

This should be a separate PR and not hold up the current work; it can go into the Later milestone.

Reply:
Sounds reasonable; I'll make an issue for that.

Reply:
Please confirm — this currently won't work at all for me (even without moving the app after starting). If we really want to push even that off to later, I guess I don't care too much, but I wanted to make sure we're explicit on _what_ we're putting into a separate PR.

Reply:
> If we really want to push even that off to later

I don't think we should push this part to Later. Users without admin rights will normally move the app to `$HOME/Applications/Rancher Desktop`, so we do need to adjust the launchctl script to the install location.

Moving the app can be addressed later.